include/config.h

```diff
@@ -8,6 +8,10 @@
 // Ruby's actual debug build does not use the AGBPrint features.
 #define NDEBUG
 
+#ifdef DEBUG_TARGET
+#undef NDEBUG
+#endif
+
 // To enable printf debugging, comment out "#define NDEBUG". This allows
```

Comment:
I wonder if the comment needs to be changed to accomodate the new debug build target?

Reply:
I dont know if `#undef``'ing NDEBUG when its not defined is UB? we could add smth like `#ifdef NDEBUG` to catch the case that it is already commented out.

Reply:
It's safe to `#undef` things that aren't defined: [cppreference](https://en.cppreference.com/w/cpp/preprocessor/replace#.23undef_directive).

Reply:
then I am not sure what should be changed about the comment. A note that the same can be achieved using debug target perhaps ?

Reply:
@Bassoonian did you want a change here still ?

Reply:
Nah it's fine, no worries! I don't feel equipped enough to approve and merge this PR, though, so I'll leave that to the Makefile gods :)